homeassistant/components/homekit/accessories.py

```diff
@@ -499,8 +499,7 @@ def async_reset(self):
             )
         )
 
-    @ha_callback
-    def async_stop(self):
+    async def stop(self):
```

Comment:
Note that this is overriding a `HAP-python` method so it shouldn't be prefixed with `async_`